### DISCUSSION TITLE
LTS of PrimeReact

### DISCUSSION THREAD

**Discussion by tweiler0309** [primefaces/community#1762] [PrimeReact]
Hi guys,
sorry I wasn't able to find any information about LTS (long term support) of PrimeReact.
Where is this kind of information published? Or when is EOL of PrimeReact 9 and PrimeReact 10 reached? 
Is there a common guideline for every major version's LTS?
Your answer is very much appriciated.
Thx and cheers
Thorsten

**Comment by melloware** [comment]
cc @gucal @nitrogenous
